content/en/events/2020/kcc2020/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Contributor Celebration"
+weight: 1
+slug: "kcc2020"
+aliases: [ "/kcc2020" ]
+description: |
+   It's been a rough year, and without any in person events we've lost the best
+   part of the Contributor Summits - the Hallway Track. The Kubernetes Contributor
+   Celebration is an attempt to reclaim that and celebrate our accomplishments.
+   It's a time for us to relax, chat and do something fun with your fellow
+   contributors!
+type: docs
+---
+
+{{% alert title="Notice" color="warning" %}}
+This event has passed. Recordings from the event can be found on the
+[Kubernetes Community YouTube Channel](https://youtube.com/playlist?list=PL69nYSiGNLP0MPeiM9oAffsxtrUgUAu1g).
+{{% /alert %}}
+
+<img align="center" alt="you're invited" style="max-width:50%;" src="/events/past-events/2020/kcc2020/invited.png">
```

Comment:
```suggestion
<img align="center" alt="you're invited" style="max-width:50%;" src="/events/2020/kcc/invited.png">
```

This is going to be the more annoying part of the migration, ALL the links have to be updated. -_- I can also take care of this and push to your branch once the files are in the new location 👍 

we can also drop the 2020 from this directory title as its now a native part of the url.

Reply:
I just saw it, there are lots of links that needs to be updated :0